server/src/test/java/org/spine3/server/stand/StandShould.java

```diff
@@ -511,15 +511,15 @@ public void onCompleted() {
 
     @Test
     public void retrieve_collection_fields_if_required() {
-        requestSampleCustomer(new int[] {Customer.FAMILYINFO_FIELD_NUMBER - 1}, new StreamObserver<QueryResponse>() {
+        requestSampleCustomer(new int[] {Customer.NICKNAMES_FIELD_NUMBER - 1}, new StreamObserver<QueryResponse>() {
```

Comment:
Why shouldn't this test pass?
The `StreamObserver` methods are called in `Stand.execute` method and are reachable from debugger.